### ISSUE TITLE
Feed filters (included/excluded)

### ISSUE THREAD

**Issue by serrq** [KoalaSat/nostros#252]
I would like to set rules and filters over the Global feed like "show me only English speaking posts", or "hide the Chinese speaking posts", or "show only profiles with linked lightning wallets" or "show me only profiles with avatar set" or "show me profile with NIP-05". 

Of course also a combination among these.

**Comment by KoalaSat** [comment]
This is a good feedback thanks! @FrontendCrypto I don't think there is a way to detect the language of texts without using 3rd parties (and draining the battery), do you have any suggestion?

**Comment by KoalaSat** [comment]
Yes it's really complicated, I know certain clients are starting to use auto translations, despite as personal opinion I consider it really wrong, it also increase the (already inflated) battery drain of a nostr client.

**Comment by trader173** [comment]
it has been a long time since i did chinese on the computor, but we used to have to include a font that supported the type of characters we needed.    eg, simplified / traditional etc.    might this help?
73 de ki4die

**Comment by KoalaSat** [comment]
This actually a hot topic on nostr, people are exploring different ways to achieve this but still there is no consensus
